content/cookbook/development/using-mainnet-accounts-programs.md

```diff
@@ -6,15 +6,15 @@ description:
   environment.
 ---
 
-Oftentimes, local tests rely on programs and accounts available only on mainnet.
+Oftentimes, local tests rely on programs and accounts that are not available on the local validator by default.  
 The Solana CLI allows to both:
 
 - Download Programs and Accounts
 - Load Programs and Accounts to a local validator
 
 ### How to load accounts from mainnet
 
-It is possible to download the SRM token mint account to file:
+It is possible to download the JUP token mint account to file:
```

Comment:
@nickfrosty Same here - these were Serum references, Ayush converted them to JUP - maybe we should use openbook since it's OSS?